components/support/ktx/src/main/java/mozilla/components/support/ktx/kotlin/JSON.kt

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.ktx.kotlin
+
+import org.json.JSONArray
+import org.json.JSONObject
+import java.util.TreeMap
+
+@Suppress("UNCHECKED_CAST")
+fun <T> JSONArray?.toList(): List<T> {
```

Comment:
Let's move them into separate files. In the ktx component we replicate the packages and names of the classes we extend. So for example the extensions for JSONArray should be in *mozilla.components.support.ktx*.**android**.*org.json.JSONArray*